released/packages/coq-iris-string-ident/coq-iris-string-ident.0.1.0/opam

```diff
@@ -0,0 +1,34 @@
+opam-version: "2.0"
+name: "coq-iris-string-ident"
+maintainer: "Ralf Jung <jung@mpi-sws.org>"
+authors: "Tej Chajed <tchajed@mit.edu>"
+license: "BSD"
+homepage: "https://gitlab.mpi-sws.org/iris/string-ident"
+bug-reports: "https://gitlab.mpi-sws.org/iris/string-ident/issues"
+dev-repo: "git+https://gitlab.mpi-sws.org/iris/string-ident.git"
+
+synopsis: "Add support for Gallina names in intro patterns to the Iris Proof Mode"
+description: """
+This package implements string_to_ident in Ltac2, enabling support for Gallina names in Iris intro patterns (within strings).
+"""
+
+depends: [
+  "coq" {>= "8.11"}
+  "coq-iris" {>= "dev.2020-04-07.0.86b62616" | (>= "3.3.0" & < "4.0") | = "dev"}
```

Comment:
```suggestion
  "coq-iris" {(>= "3.3.0" & < "4.0") | = "dev"}
```
I'm already not a big fan of having `dev` constraints in `released` packages. Having constraints for versions that don't even live in a Coq OPAM repo is unfortunately over my threshold. My view is that we can't support third-party version naming schemes, but I'd be willing to concede having `>= "dev"`, since `dev` is basically already a carte blanche.

Reply:
Yeah this seems fine; we likewise have `coq-iris` in this repo depend on a released coq-stdpp. I expect people to use either the packages from this repo or the one from ours, but not mix-and-match.

Reply:
FWIW ever since I learned about `--ignore-constraints-on` I am also no longer strongly attached to having the `| = "dev"` in this repo (but we still definitely want them in ours, even parts of our CI rely on that).